project/tests_codeception/Acceptance/Test03_LoginCest_rememberMeCest.php

```diff
@@ -14,12 +14,12 @@ public function login(AcceptanceTester $I)
         $I->amOnPage("/");
         $I->fillField("email", "first.name@email.com");
         $I->fillField("password", "first");
-        $I->checkOption("#remember_me");
+        #$I->checkOption("#remember_me");
```

Comment:
Obecnie ten test nic nie sprawdza, więc można go po prostu usunąć. Można go też zostawić i omówić tę sprawę na daily. Chyba możemy sobie na razie pozwolić, aby nasz produkt nie przechodził wszystkich testów, a w kolejnych sprintach zająć się to sprawą, jeśli uznamy to za istotne, bądź usunąć ten feature zupełnie, wtedy też i test.